x-pack/plugins/spaces/public/plugin.tsx

```diff
@@ -61,11 +63,11 @@ export class SpacesPlugin implements Plugin<SpacesPluginSetup, SpacesPluginStart
       getActiveSpace: () => this.spacesManager.getActiveSpace(),
     };
 
-    if (plugins.home) {
+    if (plugins.home && !this.isServerless) {
       plugins.home.featureCatalogue.register(createSpacesFeatureCatalogueEntry());
     }
 
-    if (plugins.management) {
+    if (plugins.management && !this.isServerless) {
```

Comment:
questions: Should the `spaceSelectorApp.create(...)` call below also be in a `!isServerless` block?

Reply:
@kc13greiner I think you're right. I added the check just below this

Reply:
@kc13greiner Absolutely! Just updated!

Reply:
The spaces selector app exposes a route that can be accessed to view all spaces and to select a different space. 

This is only linked to from the space selector and when deleting the current space as far as I can see so I think this can be safely disabled on serverless.